CMakePresets.json

```diff
@@ -33,15 +32,14 @@
                 "strategy": "external"
             },
             "cacheVariables": {
-                "ENABLE_CPPCHECK_DEFAULT": "FALSE",
-                "ENABLE_CLANG_TIDY_DEFAULT": "FALSE"
+                "ENABLE_CPPCHECK_DEFAULT": false,
+                "ENABLE_CLANG_TIDY_DEFAULT": false
             }
         },
         {
-            "name": "conf-unixlike-common",
-            "description": "Unix-like OS settings for gcc and clang toolchains",
+            "name": "unixlike",
+            "description": "Unixlike OS settings for gcc and clang toolchains",
             "hidden": true,
-            "inherits": "conf-common",
             "condition": {
                 "type": "inList",
```

Comment:
this is to restrictive! why dit you ignore my patch?

Reply:
there was a PR for macOS by @oskidan just a week ago were we discussed this condition https://github.com/cpp-best-practices/cpp_boilerplate_project/pull/16#issuecomment-1082111849. However, I am open to suggestions.